ui/js/reactselect.js

```diff
@@ -7,13 +7,28 @@ var ReactSelectComponent = React.createClass({
         side: React.PropTypes.string.isRequired,
         filter: React.PropTypes.string
     },
+    componentDidMount: function() {
+        var keyShortcuts = [
+            // Shortcut: select all
+            ['ctrl+a', function(ev) {
```

Comment:
Oops, didn't even think of that - used mousetrap's `mod` shorthand to cover both - thanks!